### PR TITLE
Slng output color

### PR DESCRIPTION
**Pull request by psyduc**
Add some color to Word, Definition, and Score. 
Also made those bold.

Add a variable resultsToDisplay for later to add an option to display number of results

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,22 +28,24 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
+      var magenta = 'ffb3ff';
```
Comment (rainrivas) [inline index.js]: put the `#` and `\` in the variable. Also set it to `const` since we won't be changing it.

File: index.js [hunk 2]
```diff
@@ -28,22 +28,24 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
```
Comment (rainrivas) [inline index.js:31]: We could also set this to a `const`

File: index.js [hunk 3]
```diff
@@ -28,22 +28,24 @@ program
...
+      var magenta = 'ffb3ff';
 
       console.log('='.repeat(consoleWidth()));
 
       if (results.length === 0) {
         console.log(chalk.red('No results were found, please try another phrase'));
       } else {
-        if (results.length > 3) {
-          trimRes = results.slice(0, 3);
+        if (results.length > resultsToDisplay) {
```
Comment (rainrivas) [inline index.js:39]: Add some notes to your PR stating that you're laying the ground work for the results filtering

**Review by rainrivas — CHANGES_REQUESTED** [review]
Fix the magenta string and add some notes for results

File: index.js (hunk 2 above)
Comment (psyduc) [reply index.js:31]: this was meant to change though, depending on if the option flag was entered

Comment (rainrivas) [reply index.js:31]: hmm... i think when we actually implement the number of results option we might not even need this var... either way is cool with me at this point

Comment (jwu910) [reply index.js:31]: Scope creep but im cool with it too.

File: index.js [hunk 4]
```diff
@@ -28,22 +28,24 @@ program
...
+          trimRes = results.slice(0, resultsToDisplay);
         } else {
           trimRes = results;
         }
         trimRes.forEach((result) => {
           if (typeof (result.definition) !== undefined) {
-            console.log('Word: ' + chalk.magenta(result.word));
-            console.log('Definition: ' + chalk.magenta(result.definition));
-            console.log('Score: ' + (result.thumbs_up - result.thumbs_down));
+            console.log(chalk.bold.hex('#' + magenta + '\'')('Word: ') + (result.word));
```
Comment (jwu910) [inline index.js]: Do we need `(result.word)` wrapped in parens?

File: index.js (hunk 1 above)
Comment (jwu910) [inline index.js]: If we change `FFB3FF` to anything else, this is no longer magenta, lets use a more generic name for the color.

**Review by jwu910 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (rainrivas) [reply index.js]: But decided on magenta so does this matter?

Comment (jwu910) [reply index.js]: I think it does. If we decide to theme the colors any differently, or add in the options for themed outputs in the future, it's another change we'd have to make. 

I'd suggest like titleColor, headerColor, or something along those lines.

Thoughts?

Comment (rainrivas) [reply index.js]: sure. should be done

**Review by rainrivas — APPROVED** [review]

**Pull request merged**